packages/rs-drive-abci/src/execution/validation/state_transition/common/asset_lock/transaction/fetch_asset_lock_transaction_output_sync/v0/mod.rs

```diff
@@ -13,6 +13,8 @@ use dpp::prelude::{AssetLockProof, ConsensusValidationResult};
 use dpp::validation::ValidationResult;
 use dpp::version::PlatformVersion;
 
+const MINIMAL_ASSET_LOCK_VALUE: u64 = 120000;
```

Comment:
This value needs to be updateable, so have it based on the platform version.

Reply:
Fixed